### PR TITLE
Add displayName before React.memo for React devtools

### PR DESCRIPTION
**Pull request by rolandszoke**
https://github.com/RisingStack/react-easy-state/issues/93

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -407,6 +407,32 @@ This is not necessary if you use React Router 4.4+. You can find more details an
...
+<details>
+<summary>Usage with React Developer Tools.</summary>
+<p></p>
+
+If you want React Developer Tools to recognize your reactive view components with names, you have to pass a **named component** to the `view` wrapper function instead of an anonymous one.
+
+```jsx
+import React from 'react';
+import { view, store } from 'react-easy-state';
+import Table from 'rc-table';
```
Comment (solkimicreb) [inline README.md]: We don't need `rc-table` an `cloneDeep` in the example.

File: README.md [hunk 2]
```diff
@@ -407,6 +407,32 @@ This is not necessary if you use React Router 4.4+. You can find more details an
 </details>
 <p></p>
 
+<details>
+<summary>Usage with React Developer Tools.</summary>
+<p></p>
+
+If you want React Developer Tools to recognize your reactive view components with names, you have to pass a **named component** to the `view` wrapper function instead of an anonymous one.
```
Comment (solkimicreb) [inline README.md]: This is not 100% clear. You state `named function` in the description and have an anonymous function (with inferred name) in the example code. I would probably change it to:

---

If you want React Developer Tools to recognize your reactive view components' names, you have to pass either a **named function** or an anonymous function with **name inference** to the `view` wrapper.

```jsx
import React from 'react';
import { view, store } from 'react-easy-state';

const user = store({
  name: 'Rick',
});

// don't do this (anonymous function)
export default view(() => (
  <div>{user.name}</div>
));

// do this (function with name inference)
const componentName = () => (
  <div>{user.name}</div>
);
export default view(componentName);

// or do this (named function)
function componentName () {
  return <div>{user.name}</div>
};
export default view(componentName);
```

---

Does it make sense?

Comment (rolandszoke) [reply README.md]: it makes :)

**Pull request merged**